src/Cryptol/Eval/Generic.hs

```diff
@@ -1734,7 +1734,7 @@ errorV sym _ty msg =
 --   and return the associated character, if it is concrete.
 --   Otherwise, return a '?' character
 valueToChar :: Backend sym => sym -> GenValue sym -> SEval sym Char
-valueToChar sym (VWord 8 wval) =
+valueToChar sym (VWord wval) =
```

Comment:
This change appears to have changed the semantics of this function slightly, as it now matches _any_ word size instead of just 8-bit words. Perhaps the pattern guards below should check that the `wordValWidth` is 8?